be/src/exec/parquet_schema_builder.cpp

```diff
@@ -217,4 +223,42 @@ static Status get_parquet_type_from_map(const ::parquet::schema::NodePtr& node,
     return Status::OK();
 }
 
+/*
+try to infer struct type from group node.
+
+group <name> {
+    type field0;
```

Comment:
Where is the definition from?

Reply:
parquet does not have struct type, there is no struct definition in parquet, so i use `infer struct type`.

Reply:
add comment